File: src/app/models/GistDetailModel.ts
import { ForkDetailModel } from "./ForkDetailModel";

export class GistDetailModel {
    public id: string;
    public owner: string;
    public filename: string;
    public description: string;
    public languagetag: string;
    public rawurl: string;
    public forksList: ForkDetailModel[];
}

File: GistWebApi/Models/GistDetailModel.cs
using System.Collections.Generic;

namespace GistWebApi
{
  public class GistDetailModel
  {
    public string Id { get; set; }
    public string Filename { get; set; }
    public string Languagetag { get; set; }
    public string Rawurl { get; set; }
    public IEnumerable<ForkDetailModel> ForksList { get; set; }
  }
}
